code/obj/flock/furniture.dm

```diff
@@ -284,3 +292,11 @@
 		animate_flock_passthrough(mover)
 		return 1
 	return ..()
+
+/obj/grille/flock/special_desc(dist, mob/user)
+	if(isflock(user))
+		return {"<span class='flocksay'><span class='bold'>###=-</span> Ident confirmed, data packet received.
+		<br><span class='bold'>ID:</span> Reinforced Conduit
```

Comment:
Sorry for the late review, I'd say change this to "Reinforced Barricade" instead of conduit.